lib/beaker/dsl/helpers.rb

```diff
@@ -23,14 +24,12 @@ module DSL
     #
     #
     module Helpers
-      include Beaker::DSL::Helpers::FacterHelpers
       include Beaker::DSL::Helpers::HostHelpers
-      include Beaker::DSL::Helpers::PuppetHelpers
       include Beaker::DSL::Helpers::TestHelpers
-      include Beaker::DSL::Helpers::TKHelpers
       include Beaker::DSL::Helpers::WebHelpers
       include Beaker::DSL::Helpers::HoconHelpers
       include Beaker::DSL::Helpers::Hiera
+      include BeakerPuppet::Helpers
```

Comment:
different pattern. 
should we use a better module naming scheme for beaker-hiera?
we probably shouldn't care much as hiera as a gem is going away (it's internal to puppet now)